### PR TITLE
Saving my edit…

### PR DESCRIPTION
**Pull request by chikamichi**
Let's replace that, that was coming from the commit, with something explaining _why_ I'm making this PR.

I do that, because… well you told me so :monkey:

### CODE REVIEW

**Comment by chikamichi** [comment]
Back in the PR!

It's a place to discuss code.

**Comment by chikamichi** [comment]
You may join the discussion! (One can edit the messages, btw)

**Comment by chikamichi** [comment]
You may notify someone by typing its @kusera name => sends an email notification.

**Comment by chikamichi** [comment]
Because you are the owner of the project, I guess you already are notified on any activity, new PRs, etc.

**Comment by ghost** [comment]
@chikamichi testing 1 2 3

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-Bloc Jams Assignment 5 - Basic HTML Structure
\ No newline at end of file
+Bloc Jams Assignment 5 - Basic HTML Structure
```
Comment (ghost) [inline README.md:1]: I'm making a comment on this @chikamichi.

Comment (chikamichi) [inline README.md:1]: Fair enough, I'll fix that and push to the PR again. Gimme a minute.

Comment (chikamichi) [inline README.md:1]: @kusera done!

**Comment by chikamichi** [comment]
!!!! :+1: thx :heart:

**Pull request merged**